system/modules/comments/classes/Comments.php

```diff
@@ -603,7 +618,7 @@ public static function notifyCommentsSubscribers(\CommentsModel $objComment)
 
 		while ($objNotify->next())
 		{
-			// Don't notify the commentor about his own comment
+			/** @var \CommentsNotifyModel $objNotify */
```

Comment:
Yes, it is necessary. The method returns a model collection but the IDE needs to understand that we are accessing the model.